knotx-performance-tests/jmeter-testplan/knotx.jmx

```diff
@@ -0,0 +1,1517 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
knotx.jmx - this is JMeter Test Plan file. It is generated by JMeter and we do not change it. Jmeter is under Apache License and developed under Apache Foundation.